db/db_properties_test.cc

```diff
@@ -1309,6 +1309,86 @@ TEST_F(DBPropertiesTest, EstimateNumKeysUnderflow) {
   ASSERT_EQ(0, num_keys);
 }
 
+TEST_F(DBPropertiesTest, EstimatedEarliestKeyTimestamp) {
+  std::unique_ptr<MockTimeEnv> mock_env(new MockTimeEnv(Env::Default()));
+  Options options;
```

Comment:
This test looks good, but you need to also check the property value when older files are dropped in fifo compaction.